backend/apps/common/validators.py

```diff
@@ -0,0 +1,10 @@
+from django.core.exceptions import ValidationError
+
+def validate_file_size(file, max_size_mb):
+    max_size = max_size_mb * 1024 * 1024
+    if getattr(file, "size", 0) > max_size:
+        raise ValidationError(f"Розмір файлу не може перевищувати {max_size_mb} MБ.")
+
+def validate_file_type(file, allowed_types):
+    if hasattr(file, 'content_type') and file.content_type not in allowed_types:
+        raise ValidationError(f"Файл повинен бути одним з: {', '.join(allowed_types)}.")
```

Comment:
The validator trusts file.content_type (provided by the client) to determine allowed file types. This is easily spoofable and could allow non-image files to be uploaded, potentially introducing remote code/data risks if processed or served.

Validate image content server-side (e.g., open with Pillow or use Python's imghdr/filetype), check file signature/magic bytes, and fall back to extension only if needed. Example using Pillow:
```
from PIL import Image, UnidentifiedImageError
def validate_file_type(file, allowed_types):
    try:
        Image.open(file).verify()
    except UnidentifiedImageError:
        raise ValidationError("Invalid image file.")
```